internal/rbd/rbd_util.go

```diff
@@ -703,6 +703,27 @@ func (ri *rbdImage) trashRemoveImage(ctx context.Context) error {
 	return nil
 }
 
+// DeleteTempImage deletes the temporary image created for pvc-pvc clone.
+func (rv *rbdVolume) DeleteTempImage(ctx context.Context) error {
+	tempClone := rv.generateTempClone()
+	err := tempClone.deleteImage(ctx)
+	if err != nil {
+		if errors.Is(err, ErrImageNotFound) {
+			err = tempClone.ensureImageCleanup(ctx)
+
+			return err
+		} else {
+			// return error if it is not ErrImageNotFound
+			log.ErrorLog(ctx, "failed to delete rbd image: %s with error: %v",
+				tempClone, err)
```

Comment:
lets not log here, dont pass the ctx and log it in caller if there is any error?

Reply:
> lets not log here, dont pass the ctx and log it in caller if there is any error?

ctx is definitely required for other function calls in this function.
I'll move the log upwards.

Reply:
it looks like its not addressed, can you please check this one again